packages/node/src/login/oidc/ClientRegistrar.ts

```diff
@@ -143,18 +149,30 @@ export default class ClientRegistrar implements IClientRegistrar {
       }
     );
 
+    let clientExpiresAt = 0;
+    if (
+      typeof registeredClient.metadata.client_secret_expires_at === "number"
+    ) {
+      // client_secret_expires_at is in seconds, convert it to milliseconds:
+      clientExpiresAt =
+        registeredClient.metadata.client_secret_expires_at * 1000;
+    }
+
     const infoToSave: Record<string, string> = {
       clientId: registeredClient.metadata.client_id,
       idTokenSignedResponseAlg:
         registeredClient.metadata.id_token_signed_response_alg ?? signingAlg,
     };
     if (registeredClient.metadata.client_secret) {
       infoToSave.clientSecret = registeredClient.metadata.client_secret;
+      infoToSave.clientExpiresAt = clientExpiresAt.toString();
```

Comment:
I think we may always want to store this, though technically `client_secret_expires_at` is only returned if a `client_secret` is returned: https://openid.net/specs/openid-connect-registration-1_0.html#RegistrationResponse